### ISSUE TITLE
Multiple "zombie" tip windows when dragging history graphs

### ISSUE THREAD

**Issue by tomjhou** [home-assistant/frontend#18016]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

This issue affects the mobile version.

If I drag any history graph when scrolling the UI, a tip window pops up giving detailed info, even though 99% of the time that is not my intention. I am usually scrolling to see something ELSE (i.e. _not_ the graph).

Not only that, these "zombie" tip windows are hard to dismiss. They _add_ to existing tip windows, rather than _replacing_ them (see screenshot below with 3 tip windows). And unlike normal tip windows, they can only be dismissed by clicking the SAME graph where it was first triggered. And you have to be super careful, as attempting to dismiss these windows often just causes another to pop up.

<img src="https://github.com/home-assistant/frontend/assets/2946264/6d45ca28-09ff-424a-a671-52a1d7f424c3" width="300">

### Describe the behavior you expected

While it is normal that tapping a history graph opens a tip window, a drag/swipe tap should not do this, since user is usually scrolling to go elsewhere, i.e. NOT the graph. If a tip window does pop up, it should be dismissible the normal way, by clicking anywhere on the UI, not necessarily the exact same window where it first triggered.

### Steps to reproduce the issue

1. Tap AND drag any history graph. This elicits a tool tip window that is hard to get rid of.
2. Dragging additional history graphs produces tip windows for all of them, with no apparent limit.
...

### What version of Home Assistant Core has the issue?

Latest version (as of 2024.4.3).

### In which browser are you experiencing the issue with?

Any mobile version

### Which operating system are you using to run this browser?

Android

**Comment by RobertD502** [comment]
Experiencing a similar situation (although, perhaps different since I noticed this when viewing a graph from the History section) and it has made checking specific data points on a graph viewed in the History section when on the iOS companion app a no-go.

When clicking on a data point in the graph, it opens the entity card instead of showing me the value for the selected data point. Video recording below:


https://github.com/home-assistant/frontend/assets/52541649/bab04bdc-3e1d-4709-9411-996de63fff45

As you can see, as soon as I stop scrolling through the data on the graph (raise my finger off of the screen), a new window of the entity pops up. Same behavior occurs if clicking on a single point on the graph.

**Comment by blandry3** [comment]
Same issue on 2023.10.1.

**Comment by del13r** [comment]
This has been annoying me since 2023.10
Only now have I been bothered enough to investigate if anyone else noticed it.

I was going to raise a new issue until I found this issue so I will add my 2cents.

Issue started with 2023.10
Issue occurs in both iOS app and chrome for iOS.

Here is what happens when I use my finger to scroll a page with multiple graphs on it

![image](https://github.com/home-assistant/frontend/assets/86141523/22eff5e4-3f37-4085-bf2c-fc1e13860519)

Then when I actually want to look at a particular graph, the tips shown on the previous page are carried over to the new page and are still visible making it very cluttered and confusing as they are not relevant to the isolated graph being shown.

![image](https://github.com/home-assistant/frontend/assets/86141523/2ec782e3-d808-4f2d-a323-6548ce478e5a)

**Comment by ColColonCleaner** [comment]
Similar issue here where I'm unable to reliably dismiss the popup once it's activated. Usually requires clicking on the same graph again in order to get the popup to go away.

https://github.com/home-assistant/frontend/assets/1903507/b54606b3-1d8e-4843-a46c-f77b0dd0b49f

**Comment by del13r** [comment]
Hi, is there any interest in fixing this issue?
Issue started with 2023.10
We are now in 2024.1 and the issue is still present.

**Comment by ColColonCleaner** [comment]
It was there before 2023.10 if I'm remembering correctly. There was a patch for a similar issue around that time, but not this issue.
Either way it would be great to see this fixed!

**Comment by tomjhou** [comment]
Hello, original poster here. I'm glad to see I'm not the only one bothered by this. I'm also surprised this issue is still not fixed. Also, it seems there is a whole other thread on this issue: https://github.com/home-assistant/frontend/issues/18198

**Comment by RobertD502** [comment]
bump

**Comment by nistvan86** [comment]
Is there any way to disable pointer interaction with the graph in the meantime to not accidentally trigger this?

**Comment by tomjhou** [comment]
Interesting idea, will give it a try! To clarify: I assume disabling pointers will stop normal tip windows also? Those (elicited by simple tap without dragging) are actually useful. Whereas swipe gestures (i.e. tap + drag) produce zombie windows that just won't die.

**Comment by nistvan86** [comment]
Yes this stops any tooltips from showing, as it won't register touch events on the graph anymore.
If you find a better targeted CSS selector to only hide the erroneous popus, please share :)

**Comment by melyux** [comment]
This has gotten even worse since the zoomable history graph update in 2025.1.

**Comment by tomjhou** [comment]
> This has gotten even worse since the zoomable history graph update in 2025.1.

Yikes, that is unfortunate. In what way is it worse?

Seems like the the zoomable graph update is broken in multiple ways, judging by all the new issues opened up about it in the last week. I'm worried that whoever is working on this is spending time adding new features (that barely work) rather than fixing the old issues (that also don't fully work):

https://github.com/home-assistant/frontend/issues/23795
https://github.com/home-assistant/frontend/issues/23766
https://github.com/home-assistant/frontend/issues/23720

**Comment by melyux** [comment]
@tomjhou The tooltips even more "sticky" even after I let go of the graph lines after 2025.1

**Comment by del13r** [comment]
Things seem to be much better after 2025.2 release.
Note on what changed https://www.home-assistant.io/blog/2025/02/05/release-20252/#preparing-our-graphs-for-the-future

**Comment by del13r** [comment]
Hi @tomjhou this can probably be closed now since 2025.2 was released

**Comment by tomjhou** [comment]
> Hi @tomjhou this can probably be closed now since 2025.2 was released

I am running 2025.2.5, and the issue is still there for me. Have you actually verified that it is fixed, rather than just assuming that the overhaul ought to have fixed it?

Tom

**Comment by tomjhou** [comment]
> Hi @tomjhou this can probably be closed now since 2025.2 was released

Hooray, looks like it is fixed!

Tom

**Comment by del13r** [comment]
> Hooray, looks like it is fixed!
> 
> Tom

Hi Tom, Good to hear the issue is fixed.

As you are the author of this issue, only you have the permissions to close this issue. Please close if you feel it is now fixed.
